src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,55 @@
+@import './variables';
+
+@mixin product-title-font {
+  font-family: $card-ff;
+  color: $dark-color;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), mixin names should be written in kebab-case. Consider renaming `product-title-font` to `product-title-font`.